types/victory/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Victory 0.9.3
+// Type definitions for Victory 1.0.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
@DanielRosenwasser yep, 1m.